modules/extensions/src/api/experimental/index.ts

```diff
@@ -1,3 +1,2 @@
 export * as auth from "./auth";
-export * as exec from "./exec";
```

Comment:
i'd leave an import here from the non-experimental API for now to keep this non breaking for now

```
export * as exec from "../exec";
```